test/logger.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+const expect = require('chai').expect;
+const { initKafkaLoggingProducer, writeLog } = require('../src/logger');
+const KafkaProducer = require('no-kafka');
+const sinon = require('sinon');
+const kafkaConfig = require('../src/kafkaConfig');
+
+describe('test/logger.js > ', () => {
+
+  it('Happy path:call producer with the right args, call the init function and send', () => {
```

Comment:
Seems like all these tests use some common logic. Do you think it would be better and more intuitive to extract out the common code?

Reply:
hmm the expect statements are completely different though. It can be done but would make it a little weird where I probably have to use if checks to see what expect statement to use at each place.